scripts/beachball/index.ts

```diff
@@ -9,6 +9,9 @@ export const config: BeachballConfig = {
       disallowedChangeTypes: ['major'],
     },
   ],
+  // This prevents accidental major changes. You can manually do a major change by selecting
+  // "minor" on the command line and then changing it to "major" by editing the change file.
+  disallowedChangeTypes: ['major'],
```

Comment:
unfortunately this won't work as is. We will either need to update beachball code (I don't think it's going to be straight forward) or add this individually in each package.json
It doesn't work because of `groups` is being defined above.

Reply:
```
Found changes in the following packages:
  office-ui-fabric-react
Cannot have a disallowedChangeType inside a package config (office-ui-fabric-react) when there is a group defined; use the groups.disallowedChangeTypes instead.
ERROR: Groups defined inside the configuration is invalid
[
  {
    name: 'Fluent UI React',
    include: [ 'packages/office-ui-fabric-react', 'packages/react' ],
    disallowedChangeTypes: [ 'major' ]
  }
]
```

Reply:
i believe this is the right fix: https://github.com/microsoft/beachball/pull/403/files